CHANGELOG.md

```diff
@@ -1,12 +1,18 @@
 # Changelog
 
-## Versions from 0.4x
+## v0.55.13 - 2025-09-03
 
-### Ongoing
+- Activate new Sense hysteresis based switch action via PR [326](https://github.com/plugwise/plugwise_usb-beta/pull/326)
+
```

Comment:
Please add here: 
- Link to plugwise_usb [v0.45.0](https://github.com/plugwise/python-plugwise-usb/releases/tag/v0.45.0)